### ISSUE TITLE
Update supported connectors on website

### ISSUE THREAD

**Issue by SamWoolerton** [lowdefy/lowdefy#635]
Just a heads up that you probably want to update http://lowdefy.com/ with the connectors you now support - it only shows Mongo, Google Sheets, and API calls as supported but your docs show way more!

**Comment by Gervwyk** [comment]
Ahh! Thanks for letting us know, I'll sort that out asap.

**Comment by ZaLiTHkA** [comment]
I almost did a hard pass on Lowdefy thanks to this "issue".. Investigating the use of "low code" platforms for building some BI apps at work, and one of the sources I need to use for this is SQLite. Good thing I checked the GitHub repo `README` file hey? (:

@Gervwyk, is the website source contained within this same repo, or is that handled separately? I was looking for that to make the updates and send a PR, but then I found *this* report while searching the repo for "website".

**Comment by Gervwyk** [comment]
Thanks @ZaLiTHkA. Seems like this is a more serious issue than I thought :/ 

It's currently in a separate repo, the issue I'm having is that I used Hype4 to make the hero animation, and for some reason every time I update the site (hosted on Netlify), the hero animation breaks. Seems like a build / cdn issue. So I decided to also redo the hero animation into a giff I guess. 

Was also trying to come up with something a little better if I'm going to redo it, but running low on creative juice. Any suggestions?
